src/core/api.proto

```diff
@@ -223,6 +223,26 @@ message InferRequestHeader
   //@@     request.
   //@@
   repeated Output output = 3;
+
+  //@@  .. cpp:var:: uint32 priority
+  //@@
+  //@@     The priority value of this request. If priority handling is not
+  //@@     enable for the model, then this value is ignored. The default value
+  //@@     is 0 which indicates that the request will not be prioritized.
```

Comment:
".. the request will be assigned the default priority associated with the model."